### DISCUSSION TITLE
为什么我直接拖拽start.sh文件在进终端无法启动呀

### DISCUSSION THREAD

**Discussion by Zenbukorosu** [lisai9093/YYS#29] [Q&A]
我是小白，请问按照步骤一步步来最后在通过终端运行这里卡住了怎么办，另外我是m3只能用mumu模拟器，adb怎么操作请大佬讲讲

**Comment by lisai9093** [comment]
可能是sh文件不是可执行属性。你可以在终端查看文件属性：
`ls -l start.sh`
如果属性里没有“x”就表示不能被运行，同样用终端添加即可：
`chmod +x start.sh`

**Reply by lisai9093** [reply]
另外我也是M芯片，用的是MuMu Pro模拟器。用菜单栏下的选项开启ADB：
<img width="260" alt="Screenshot 2024-12-03 at 10 14 42 PM" src="https://github.com/user-attachments/assets/beecbb00-9af6-40ae-bd95-577d4996955b">
